website/content/docs/secrets/databases/index.mdx

```diff
@@ -57,15 +57,16 @@ management tool.
    By default, the secrets engine will enable at the name of the engine. To
    enable the secrets engine at a different path, use the `-path` argument.
 
-1. Configure Vault with the proper plugin and connection information:
+2. Configure Vault with the proper plugin and connection information:
 
    ```shell
    $ vault write database/config/my-database \
        plugin_name="..." \
        connection_url="..." \
        allowed_roles="..." \
        username="..." \
-       password="..."
+       password="..." \
+       disable_escaping="true/false"
```

Comment:
nit: I feel like this might not be necessary here either since it is an "advanced feature"?

Reply:
I think that makes sense, with the paragraph at the bottom of the doc to specifically outline it instead.